packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import * as fs from 'fs';
+import * as path from 'path';
+import { SourceMapConsumer, SourceMapGenerator } from 'source-map';
+import { minify } from 'terser';
+
+const { transformAsync } = require('@babel/core');
```

Comment:
Any reason why not add `@types/babel-core`?

Reply:
It’s outdated unfortunately.  Appears to still be on 6.x (7.5 is latest).